sample/echo/server.cc

```diff
@@ -72,6 +87,7 @@ int main()
 
     // Delete closures.
     // Attention: should delete the closures after server stopped, or may be crash.
+    delete servlet;
     delete options.work_thread_init_func;
     delete options.work_thread_dest_func;
```

Comment:
servlet should first Unregistered then deleted, or becomes wild pointer and used in server.